be/src/olap/row_cursor.h

```diff
@@ -36,10 +36,13 @@ class RowCursor {
 public:
     static const int DEFAULT_TEXT_LENGTH = 128;
 
-    RowCursor();
+    RowCursor() = default;
 
     // 遍历销毁field指针
-    ~RowCursor();
+    ~RowCursor() {
+        delete [] _owned_fixed_buf;
+        delete [] _variable_buf;
```

Comment:
If _variable_buf holds a ref of an Object such as Slice,can ```delete [] _variable_buf ``` trigger Slice's destructor.

Reply:
no, because the type of _variable_buf is char*, so delete[] _variable_buf will just release the memory allocated by corresponding new[] not matter whatever the _variable_buf is used for.

Reply:
i have appended comments to explain why i do this